ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/FragmentationStrategyCollectionImpl.java

```diff
@@ -44,4 +47,10 @@ public void handleViewAddedEvent(ViewAddedEvent event) {
 		fragmentationStrategyMap.put(event.getViewName(),
 				fragmentationStrategyForView);
 	}
+
+	@EventListener
+	public void handleViewDeletedEvent(ViewDeletedEvent event) {
+		ldesFragmentRemover.removeLdesFragmentsOfView(event.getViewName());
```

Comment:
We should be cautious with this architecture. Currently we have a chain where we call remover.remove which calls repository.remove which calls mongoRepository.remove.
I wonder if the fragmentRemover could be listening to these events as well? Then it wouldn't be a simple pass through class. This method could just update his internals (the strategyMap).

Reply:
To be checked what the best approach is:
* handle events separately
* handle events in one go
Initial reasoning why the ViewDeletedEvent is handled in one go is because ViewAddedEvent is handled the same way. In that case however, there is an order that needs to be respected. We don't have that constraint in the ViewDeletedEvent